### PR TITLE
Laikad: blocking raw_gnss receive

### PR DESCRIPTION
**Pull request by haraschax**
Saves 5-10% cpu

### CODE REVIEW

File: selfdrive/locationd/laikad.py [hunk 1]
```diff
@@ -449,7 +449,7 @@ def main(sm=None, pm=None):
   laikad = Laikad(save_ephemeris=not replay, auto_fetch_navs=use_internet, use_qcom=use_qcom)
 
   while True:
-    for in_msg in messaging.drain_sock(raw_gnss_sock):
+    for in_msg in messaging.drain_sock(raw_gnss_sock, wait_for_one=True):
```
Comment (adeebshihadeh) [inline selfdrive/locationd/laikad.py:452]: this makes it such that the clocks block below will never be hit. for this case, you probably just want to do a single blocking receive

Comment (HaraldSchafer) [reply selfdrive/locationd/laikad.py:452]: You mean because of this
```
 laikad.got_first_gnss_msg
 ```
 ?

Comment (adeebshihadeh) [reply selfdrive/locationd/laikad.py:452]: yeah

Comment (HaraldSchafer) [reply selfdrive/locationd/laikad.py:452]: Not fully sure why the clocks block is there, seems redundant

Comment (HaraldSchafer) [reply selfdrive/locationd/laikad.py:452]: Let me just remove

**Pull request merged**